### PR TITLE
[Analytics] Add Network Analytics v2

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-1817.

### CODE REVIEW

File: content/analytics/network-analytics/understand/concepts.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+For more background information on Adaptive Bit Rate sampling, refer to the [Explaining Cloudflare's ABR Analytics](https://blog.cloudflare.com/explaining-cloudflares-abr-analytics/) blog post.
+
+## Edge Sample Enrichment
+
+Network Analytics v2 (NAv2) can provide accurate data due to the sample rate and to Edge Sample Enrichment.
+
+NAv2 sample rates vary depending on the mitigation service. For example:
+
+* The sample rate for dosd is 1/10,000 packets.
```
Comment (marciocloudflare) [inline content/analytics/network-analytics/understand/concepts.md]: Just checking if `dosd` isn't a typo.

Comment (marciocloudflare) [reply content/analytics/network-analytics/understand/concepts.md]: Also, if it's not, should it be between backticks?

File: content/analytics/network-analytics/understand/concepts.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* The sample rate for Magic Firewall events changes dynamically from 1/100 to 1/1,000,000 packets based on the number of packets.
+* The sample rate for flowtrackd changes dynamically from 1/100 to 1/10,000 packets based on the volume of packets.
```
Comment (marciocloudflare) [inline content/analytics/network-analytics/understand/concepts.md]:
```suggestion
* The sample rate for `flowtrackd` changes dynamically from 1/100 to 1/10,000 packets based on the volume of packets.
```

File: content/analytics/network-analytics/understand/main-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+The Network Analytics **Activity log** shows up to 100 log events — including both allowed and dropped packets — in the currently selected time range, paginated with 10 results per page per time range view (the [GraphQL Analytics API](/analytics/graphql-api/) does not have this limitation).
+
+To display event details, click the expansion widget associated with the events.
+
+## Top items
+
+The **Source country** / **Source data center**, **Source**, and **Destination** panels display the top items in each view.
+
+To see the top data centers, select _Data center_ from the drop-down list in the **Source country** view. The **Source data center** view replaces the **Source country** view.
```
Comment (marciocloudflare) [inline content/analytics/network-analytics/understand/main-dashboard.md]:
```suggestion
To display the top data centers, select _Data center_ from the drop-down list in the **Source country** view. The **Source data center** view replaces the **Source country** view.
```

File: content/analytics/network-analytics/understand/main-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+To select the number of items to display, use the drop-down list associated with the view.
+
+## TCP Flag
```
Comment (marciocloudflare) [inline content/analytics/network-analytics/understand/main-dashboard.md]:
```suggestion
## TCP flag
```

Comment (marciocloudflare) [reply content/analytics/network-analytics/understand/main-dashboard.md]: Adding lowercase here as in the text it's also lowercase

**Review by marciocloudflare — APPROVED** [review]

File: content/analytics/network-analytics/understand/concepts.md (hunk 1 above)
Comment (pedrosousa) [reply content/analytics/network-analytics/understand/concepts.md]:
```suggestion
* The sample rate for `dosd` is 1/10,000 packets.
```

File: content/analytics/network-analytics/understand/main-dashboard.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+This panel displays the TCP flags set for all the traffic currently displayed in the dashboard, including both allowed and mitigated traffic.
```
Comment (pedrosousa) [inline content/analytics/network-analytics/understand/main-dashboard.md]:
```suggestion
The **TCP Flag** panel displays the TCP flags set for all the traffic currently displayed in the dashboard, including both allowed and mitigated traffic.
```

**Pull request merged**